README.md

```diff
@@ -68,6 +68,24 @@ Quantiles
 
 ```
 
+### Usage with [`LogDensityProblems.jl`](https://github.com/tpapp/LogDensityProblems.jl)
+
+It can also be used with models defining the [`LogDensityProblems.jl`](https://github.com/tpapp/LogDensityProblems.jl) interface by wrapping it in `AbstractMCMC.LogDensityModel` before passing it to `sample`:
```

Comment:
Is it really necessary to wrap the log density? Couldn't we wrap anything in a LogDensityModel that's not one of our specialized model types? We could even check `LogDensityProblems.capabilities !== nothing` to ensure that we do not wrap anything accidentally that does not implement the interface.